workflow-templates/im-deploy-az-database.yml

```diff
@@ -30,110 +30,78 @@ on:
       environment:
-        description: The environment to deploy to - dev|qa|stage|prod|demo|uat # TODO: update for the environments you use
+        description: The environment to deploy to
         required: true
+        type: choice
+        options: # TODO: Update for the environments that are available
+          - dev
+          - qa
+          - stage
+          - demo
+          - uat
+          - prod
 
   # TODO: If a repository has multiple deployable artifacts that are deployed together through the 'Deploy Multiple Items' workflow:
   #       1. Uncomment the repository_dispatch trigger
   #       2. Delete the workflow_dispatch trigger
-  #       3. Change the env.RELEASE_TAG value from "${{ github.event.inputs.tag }}" to: "${{ github.event.client_payload.tag }}"
-  #       4. Delete the 'set-vars' job
-  #       4. Delete the 'stakeholder-approval' job
-  #       5. Delete the 'attestor-approval' job
-  #       6. Delete the 'validate-tag-is-in-main-for-prod-deploys' job
-  #       7. Replace all instances of "${{ needs.set-vars.outputs.ENVIRONMENT }}" with "${{ github.event.client_payload.environment }}"
+  #       3. Change the env.ENVIRONMENT value from "${{ github.event.inputs.environment }}" to: "${{ github.event.client_payload.environment }}"
+  #       4. Change the env.RELEASE_TAG value from "${{ github.event.inputs.tag }}" to: "${{ github.event.client_payload.tag }}"
+  #       5. Delete the 'stakeholder-approval' job
+  #       6. Delete the 'attestor-approval' job
+  #       7. Delete the 'validate-tag-is-in-main-for-prod-deploys' job
   #       8. Under the 'deploy-az-db' job, delete the needs property
   #       9. Under the 'update-deployment-board-and-send-teams-notification' job, update the needs property to be: "needs: [deploy-az-db]"
   # repository_dispatch:
   #   types: [<deployable_name>_deploy] # TODO: Replace <deployable_name>.  This will be used in the 'Deploy Multiple Items' workflow to target this deployment workflow.
 
 env:
+  ENVIRONMENT: ${{ github.event.inputs.environment }}
   RELEASE_TAG: ${{ github.event.inputs.tag }} # This is the tag that we'll be deploying
   DB_SERVER_PORT: '1433' # TODO: Verify that this is the correct port
   MIGRATIONS_FOLDER: '' # TODO: Add the path to the migrations folder
   MIGRATIONS_HISTORY_TABLE: 'MigrationHistory' # TODO: Verify this table name
   TIMEZONE: 'america/denver' # TODO: Verify timezone
 
 jobs:
-  set-vars:
-    runs-on: ubuntu-latest # Force this to run on github-hosted runner by using a tag that does not exist on self-hosted runners
-
-    outputs:
-      ENVIRONMENT: ${{ steps.clean-env.outputs.ENVIRONMENT }} # To use this value: ${{ needs.set-vars.outputs.ENVIRONMENT }}
-
-    steps:
-      - name: Checkout Repository
-        uses: actions/checkout@v2
-        with:
-          fetch-depth: 0
-
-      - name: Verify Tag Exists
-        uses: im-open/verify-git-ref@v1.0.1
-        with:
-          branch-tag-sha: ${{ env.RELEASE_TAG }}
```

Comment:
I honestly can't remember why we had this check in addition to the `is-tag-reachable-from-default-branch` check, but it seems like we're ok with deleting it now? Just double checking.

Reply:
I actually think I got a little delete-happy and it should be there.  It's to ensure someone didn't fat-finger the tag before the workflow really gets started.

Reply:
I put that back in and fixed it in a couple of other places where I got a little too excited to delete things. :)

Reply:
Ah that's right, I remember now! Glad I double checked 😄